src/test/clojure/clara/generative_tests/generators.clj

```diff
@@ -0,0 +1,102 @@
+(ns clara.generative-tests.generators
+  (:require [clojure.math.combinatorics :as combo]
+            [clara.rules :refer :all]
+            [schema.core :as sc]))
```

Comment:
nit: `s` is typically the alias used for this ns in this project and from schema docs themselves.